api/data_pipeline.py

```
@@ -382,10 +497,24 @@ def prepare_data_pipeline(is_ollama_embedder: bool = None):
         embedder_transformer = ToEmbeddings(
             embedder=embedder, batch_size=batch_size
         )
+        
+        # Test the embedder immediately to verify it works
+        try:
+            logger.info(f"Testing embedder with API key present: {bool(os.environ.get('OPENAI_API_KEY'))}")
+            test_doc = [{"text": "test embedding"}]
```

Ruff (0.11.9)

Local variable `test_doc` is assigned to but never used

Remove assignment to unused variable `test_doc`

(F841)